swap/src/database.rs

```diff
@@ -3,13 +3,39 @@ use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use std::{fmt::Display, path::Path};
 use uuid::Uuid;
 
-mod alice;
-mod bob;
+pub mod alice;
+pub mod bob;
```

Comment:
```suggestion
mod alice;
mod bob;
```

Why is this needed? I created these modules purely to nicely organise the code within the `database` module, I did not intend them to be part of the `database` module API.

Reply:
its not, I had them Alice and bob in the same file till things got messy with a rebase.